sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyOperations.cs

```diff
@@ -19,4 +19,26 @@ public enum KeyOperations : uint
 
         All = uint.MaxValue
     }
+
+    public static class KeyOperationsExtensions
+    {
+        public static KeyOperations ParseFromString(string value)
+        {
+            switch (value)
+            {
+                case "encrypt": return KeyOperations.Encrypt;
+                case "decrypt": return KeyOperations.Decrypt;
+                case "sign": return KeyOperations.Sign;
+                case "verify": return KeyOperations.Verify;
+                case "wrapKey": return KeyOperations.Wrap;
+                case "unwrapKey": return KeyOperations.Unwrap;
+                default: return KeyOperations.Other;
+            }
+        }
+
+        public static string AsString(KeyOperations keyType)
+        {
+            return (keyType).ToString();
```

Comment:
>return (keyType).ToString(); [](start = 12, length = 28)

The strings are not the same as the enumeration.  for instance wrap I'm assuming would map to wrapKey.  I think you need to implement this similiarly to the switch statement you have in ParseFromString.

Reply:
changed